recipe/install-devel.sh

```diff
@@ -1,7 +1,7 @@
 # not all packages have the license file.  Copy it from mkl, where we know it exists
-cp -f $SRC_DIR/mkl/info/LICENSE.txt $SRC_DIR
+cp -f $SRC_DIR/mkl/info/license.txt $SRC_DIR
```

Comment:
```suggestion
cp -f $SRC_DIR/mkl/info/licenses/license.txt $SRC_DIR
```

Reply:
Thank you @isuruf . I realized that. It's odd that the location only changed on unix platforms.

Reply:
The location depends on the conda-build version that the original packages were built with.